models/appModels.js

```diff
@@ -76,6 +76,34 @@ const fetchSpecificCommentsByArticleId = (id) => {
     })
 }
 
-module.exports = {fetchTopics, fetchArticles, fetchSpecificArticle, fetchSpecificComments, fetchSpecificCommentsByArticleId}
+const publishCommentWithArticleId = (article_id, body) => {
+    const newComment = [
+        body.comment,
+        article_id,
+        body.username,
+        
+    ]
+
+
+    const bodySize = Object.keys(body).length
+    if (bodySize > 2){
+        return Promise.reject({status: 400, msg: 'Bad Request'})
+    }
+
+
+
+    const queryString = `INSERT INTO comments (body, article_id, author) VALUES ($1, $2, $3) RETURNING *; `
+
+     return db.query(queryString, newComment).then(() => {
+        return db.query(`SELECT * FROM comments;`).then(({rows}) => {
```

Comment:
you wouldn't need to do another database request here as you're RETURNING from from the previous one